generators/ci-cd/index.js

```diff
@@ -46,6 +46,13 @@ module.exports = class extends BaseGenerator {
             defaults: false,
             description: 'Automatically configure Jenkins'
         });
+
+        // Automatically configure Jenkins
```

Comment:
Let's remove confusion and correct the comment😁

Reply:
Done!